src/payment/strategies/klarna/klarnav2-payment-initialize-options.ts

```diff
@@ -0,0 +1,28 @@
+import { KlarnaLoadResponse } from './klarna-payments';
+
+/**
+ * A set of options that are required to initialize the KlarnaV2 payment method.
+ *
+ * When KlarnaV2 is initialized, a list of payment options will be displayed for the customer to choose from.
+ * Each one with its own widget.
+ */
+export default interface KlarnaV2PaymentInitializeOptions {
+    /**
+     * The ID of a container which the payment widget should insert into.
+     */
+    container: string;
+    /**
+     * The payment_method_category specifies which of Klarna’s customer offerings
+     * (Pay now, Pay later or Slice it) that is being shown.
+     */
+    payment_method_category: string;
```

Comment:
As far as I know, this is just `methodId`, which you already have access to in the `initialize` method. Therefore you should be using that instead of passing a new parameter that essentially carries the same information.